app/authentication/admin.py

```diff
@@ -0,0 +1 @@
+# Register your models here.
```

Comment:
register your model for it to be displayed in the admin panel

Reply:
Please check if I have done it the right way